cryptography/src/bls12381/dkg/ops.rs

```diff
@@ -68,21 +85,15 @@ pub fn verify_commitment<V: Variant>(
     if commitment.degree() != t - 1 {
         return Err(Error::CommitmentWrongDegree);
     }
-    Ok(())
+    Ok(VerifiedCommitment::new(commitment))
 }
 
 /// Verify that a given share is valid for a specified recipient.
-pub fn verify_share<V: Variant>(
```

Comment:
nit: for both `verify_commitment` and `verify_share`, is it better to make them methods on `poly::Public<V>` and `VerifiedCommitment` respectively? Or leave them as floating functions as they are now?

Reply:
Definitely a good question. I prefer to keep this PR scoped (it can rapidly turn into a large refactor if we aren't careful).